cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -313,11 +319,11 @@ describe('CourseOutlinePage', function() {
             'staff-lock-editor', 'unit-access-editor', 'discussion-editor', 'content-visibility-editor',
             'settings-modal-tabs', 'timed-examination-preference-editor', 'access-editor',
             'show-correctness-editor', 'highlights-editor', 'highlights-enable-editor',
-            'course-highlights-enable'
+            'course-highlights-enable', 'optional-completion-editor'
         ]);
         appendSetFixtures(mockOutlinePage);
         mockCourseJSON = createMockCourseJSON({}, [
-            createMockSectionJSON({}, [
+            createMockSectionJSON({completion_tracking_enabled: true}, [
```

Comment:
@DanielVZ96, the completion tracking should be enabled for the course instead of the section.

Also, you should be able to use something like `console.warn(window. course)` to verify that it's applied correctly.